pest/src/inputs/mod.rs

```diff
@@ -7,14 +7,8 @@
 
 //! A `mod` containing the `Input`-related constructs.
```

Comment:
This will need to be amended to its new name/purpose.

Reply:
:+1: